web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +135,40 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    this.svg.select("circle").remove();
+    let circleData = {};
+
+    if (!_.isEmpty(this.props.data) && this.props.showFlash) {
```

Comment:
should the `this.props.showFlash` check wrap this entire function? or do we always want to call `this.svg.select("circle").remove();` just to be safe?